.github/workflows/main.yaml

```diff
@@ -103,6 +75,23 @@ jobs:
           CODE_QUALITY: ${{ env.PASSED_PERCENTAGE }}
           BADGE_PATH: .github/badges/passed_percentage.svg
 
+      # Get the number_of_tests from pytest_scores.json
+      - name: Obtain total_tests from JSON
+        if: '!cancelled()'
```

Comment:
Should all of these be removed?

Reply:
Nope, this is used to continue execution even if pylint or pytest fails